src/FapiMemberPlugin.php

```diff
@@ -615,21 +640,21 @@ public function fapiApi() {
 	 */
 	public function getFapiClients() {
 		if ( $this->fapiClients === null ) {
-			$apiUser = get_option( self::OPTION_KEY_API_USER, null );
-			$apiKey  = get_option( self::OPTION_KEY_API_KEY, null );
+			//$apiUser = get_option( self::OPTION_KEY_API_USER, null );
+			//$apiKey  = get_option( self::OPTION_KEY_API_KEY, null );
 			$apiUrl  = get_option( self::OPTION_KEY_API_URL, 'https://api.fapi.cz/' );
-			// $fapiCredentials = get_option( self::OPTION_KEY_API_CREDENTIALS, null );
+			$fapiCredentials = get_option( self::OPTION_KEY_API_CREDENTIALS, null );
 
-			// if ( $fapiCredentials === null ) {
-				$fapiCredentials = array(
+			 if ( empty( $fapiCredentials ) ) {
+				 $fapiCredentials = array(
 					array(
-						'username' => $apiUser,
-						'token'    => $apiKey,
+						'username' => null,
+						'token'    => null,
 					),
 				);
-				// } else {
-				// $fapiCredentials = json_decode( $fapiCredentials, true );
-				// }
+				} else {
+				 $fapiCredentials = json_decode( $fapiCredentials, true );
+				}
```

Comment:
Bude tohle zpětně kompatibilní? že v případě že záakzník bude mít hodnoty uložené pouze v self::OPTION_KEY_API_USER a self::OPTION_KEY_API_KEY to bude fungovat? 
Klient může mít i starší verzi WP.